Provide access to the measurement adapter

sven1103 opened the pull request:
This commit allows the client to access the measurement
adapter used in the Oxford Nanopore Measurement.

As the adapter field is an optional metadata property, it will
be an empty String if the field was not set.

wow-such-code reviewed (dismissed):
looking good

KochTobi commented on src/test/groovy/life/qbic/datamodel/datasets/datastructure/OxfordNanoporeMeasurementSpec.groovy:
```diff
@@ -193,4 +193,32 @@ class OxfordNanoporeMeasurementSpec extends Specification {
 
     }
 
+    def "missing adapter metadata should be allowed but return empty String"() {
```
```suggestion
    def "empty String is returned for missing adapter metadata"() {
```

KochTobi reviewed (dismissed):
I would rename the test.

sven1103 committed:
Provide access to the measurement adapter

This commit allows the client to access the measurement
adapter used in the Oxford Nanopore Measurement.

As the adapter field is an optional metadata property, it will
be an empty String if the field was not set.

KochTobi reviewed (approved):
looks good
